src/app/chefs/UpdateItem.jsx

```diff
@@ -11,6 +10,7 @@ class UpdateItem extends React.Component {
       name: null,
       price: null,
       description: null,
+      item: this.props.item, // do not change = this.props.item.item
```

Comment:
this is interesting, it seems you do not need a "this" in front of these things? A syntax I'm not familiar with? Do you have a sentence or two  or a link that would take you 60s or less to include for my learning? If not, no worries.

Reply:
Oh yeah, it doesn't need "this". My comment was in reference to eslint saying to destructure--destructuring it further was breaking it. I'll update.